app/models/manageiq/providers/lenovo/physical_infra_manager/physical_server/remote_console.rb

```diff
@@ -1,23 +1,8 @@
 module ManageIQ::Providers::Lenovo::PhysicalInfraManager::PhysicalServer::RemoteConsole
   extend ActiveSupport::Concern
 
-  def remote_console_acquire_resource_queue(userid)
-    task_opts = {
-      :action => "Acquiring remote console file or url from a physical server with uuid #{ems_ref} for user #{userid}",
-      :userid => userid
-    }
-
-    queue_opts = {
-      :class_name  => self.class.name,
-      :instance_id => id,
-      :method_name => 'remote_console_acquire_resource',
-      :priority    => MiqQueue::HIGH_PRIORITY,
-      :role        => 'ems_operations',
-      :zone        => my_zone,
-      :args        => [userid, MiqServer.my_server.id]
-    }
-
-    MiqTask.generic_action_with_callback(task_opts, queue_opts)
+  included do
+    supports :console
```

Comment:
why was the `supports :console` removed from Lenovo?  Does it not work for this provider?

Reply:
Hey @chessbyte !

It wasn't removed the `remote_console_acquire_resource_queue` method was moved into core since it isn't actually lenovo specific, only the `remote_console_acquire_resource` is.  This means if another physical infra provider wants to implement a console method they don't also have to copy&paste the `remote_console_acquire_resource_queue` method.